__tests__/app.test.js

```diff
@@ -50,7 +50,7 @@ describe('GET /api/', () => {
     })
 })
 
-describe('GET /api/articles/:article_id', () => {
+describe.only('GET /api/articles/:article_id', () => {
```

Comment:
Please make sure to remove`.only`'s before PRs so you're sure all your tests are passing
Before merging make sure your tests all work 🧪